webpack.config.js

```diff
@@ -0,0 +1,69 @@
+const path = require('path');
+const HtmlWebPackPlugin = require('html-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const FixStyleOnlyEntriesPlugin = require('webpack-fix-style-only-entries');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+const WorkboxPlugin = require('workbox-webpack-plugin');
+const CopyPlugin = require('copy-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    main: './src/js/index.js',
+    styles: './src/css/index.css',
+  },
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: '[name].bundle.js',
+  },
+  module: {
+    rules: [
+      {
+        test: '/.js$/',
+        exclude: /node_modules/,
+        loader: 'babel-loader',
+      },
+      {
+        test: /\.css$/,
+        exclude: /(node_modules)/,
+        use: [MiniCssExtractPlugin.loader, 'css-loader'],
+      },
+      {
+        test: /\.(png|svg|jpg|gif)$/,
+        use: ['file-loader'],
+      },
+      {
+        test: /\.(woff|woff2|eot|ttf|otf)$/,
+        use: ['file-loader'],
+      },
+      {
+        test: /\.html$/i,
+        use: ['html-loader'],
+      },
+    ],
+  },
+  plugins: [
+    new MiniCssExtractPlugin({ filename: '[name].css' }),
+    new FixStyleOnlyEntriesPlugin(),
+    new OptimizeCSSAssetsPlugin({}),
+    new WorkboxPlugin.GenerateSW({
+      clientsClaim: true,
+      skipWaiting: true,
+    }),
+    new HtmlWebPackPlugin({
+      template: './src/views/index.html',
+      filename: './index.html',
+    }),
+    new CleanWebpackPlugin({
+      // Simulate the removal of files
+      dry: true,
+      // Write Logs to Console
+      verbose: true,
+      // Automatically remove all unused webpack assets on rebuild
+      cleanStaleWebpackAssets: true,
+      protectWebpackAssets: false,
+    }),
+    new CopyPlugin([{ from: './src/assets', to: './assets' }]),
```

Comment:
The API is changed. see https://www.npmjs.com/package/copy-webpack-plugin

```js
    new CopyPlugin({
      patterns: [
        { from: './src/assets', to: './assets' },
      ],
    }),
```

Reply:
Thanks for the review. I have been having a problem with this and your solution solved it for me